gnc/fam/src/fam_component.cc

```diff
@@ -19,24 +19,20 @@
 #include <fam/fam.h>
 
 #include <ff_common/init.h>
-#include <ff_util/ff_nodelet.h>
-
-#include <nodelet/nodelet.h>
-#include <pluginlib/class_list_macros.h>
+#include <ff_util/ff_component.h>
 
 #include <memory>
 
 namespace fam {
 
-class FamNodelet : public ff_util::FreeFlyerNodelet {
+class FamComponent : public ff_util::FreeFlyerComponent {
  public:
-  FamNodelet() : ff_util::FreeFlyerNodelet(NODE_FAM, true) {}
-  ~FamNodelet() {}
+  explicit FamComponent(const rclcpp::NodeOptions & options) : ff_util::FreeFlyerComponent(options, NODE_CTL) {}
```

Comment:
Is this supposed to be NODE_CTL or NODE_FAM?

Reply:
good catch!